pages/components/card/card.css

```diff
@@ -87,11 +116,25 @@
 .card-badge {
 	border-radius: 0 1rem 1rem 0;
 	position: absolute;
-	top: 5rem;
+	top: 10px;
 	left: 0;
 	padding: 0.5rem;
-	width: max-content;
+	max-width: 70%;
+	overflow: hidden;
 	color: var(--colour-black-one);
 	background-color: var(--colour-warning);
 	transition: 0.3s ease;
 }
+
+.card:hover .card-badge {
+	opacity: 0.4;
+	top: -30px;
+}
```

Comment:
use values in rem also in card-badge component you have use 10px i think it must be 1 rem.

Reply:
Used it to position so it will change according to the font size which I don't want I want it to be fixed.